src/js/game/map_chunk.js

```diff
@@ -213,7 +213,7 @@ export class MapChunk {
                 this.internalGenerateRandomSubShape(rng, weights),
                 this.internalGenerateRandomSubShape(rng, weights),
                 this.internalGenerateRandomSubShape(rng, weights),
-                this.internalGenerateRandomSubShape(rng, weights),
+                this.internalGenerateRandomSubShape(rng, weights)
```

Comment:
It may be better to keep the trailing comma, to keep consistent format (also I think lint requires it)